content/doc/book/pipeline/syntax.adoc

```diff
@@ -1161,11 +1161,15 @@ Example: `when { buildingTag() }`
 changelog:: Execute the stage if the build's SCM changelog contains a given regular expression pattern,
 for example: `when { changelog '.*^\\[DEPENDENCY\\] .+$' }`
 
-changeset:: Execute the stage if the build's SCM changeset contains one or more files matching the given string or glob.
+changeset:: Execute the stage if the build's SCM changeset contains one or more files matching the given pattern.
 Example: `+when { changeset "**/*.js" }+`
 +
-By default the path matching will be case insensitive, this can be turned off with the `caseSensitive` parameter,
-for example: `when { changeset glob: "ReadMe.*", caseSensitive: true }`
+The optional parameter `comparator` may be added after an attribute
+to specify how any patterns are evaluated for a match:
+`EQUALS` for a simple string comparison,
+`GLOB` (the default) for an ANT style path glob case insensitive, this can be turned off with the `caseSensitive` parameter, or
```

Comment:
This has been added after the review, as this particular parameter is enabled for the GLOB comparator, although let's wait for the implementation itself in case it changes